integration/simulation/p2p/in_mem_obscuro_client.go

```diff
@@ -130,7 +135,9 @@ func (c *inMemObscuroClient) getTransactionByHash(result interface{}, args []int
 	}
 
 	// GetTransactionByHash returns string pointer, we want string
-	*result.(*interface{}) = *encryptedResponse
+	if encryptedResponse != nil {
```

Comment:
An exception could happen if the call returned `nil`, and we then tried to deref the `nil` to return it.